src/main/java/com/fmi/cardealership/controller/PaymentController.java

```diff
@@ -67,12 +75,25 @@ public void updatePayment(@PathVariable Long id, @RequestBody @Valid PaymentDto
         logger.info(String.format("User with id %d was updated", paymentDto.getId()));
     }
 
-    private static void createTypeMapper() {
+    private void createTypeMapperEntityToDto() {
         TypeMap<Payment, PaymentDto> typeMap = modelMapper.createTypeMap(Payment.class, PaymentDto.class);
         typeMap.addMapping(Payment::getAmount, PaymentDto::setAmount);
         typeMap.addMapping(Payment::getId, PaymentDto::setId);
+        typeMap.addMapping(Payment::getDate, PaymentDto::setDate);
+
         typeMap.addMappings(mapper -> mapper.map(src -> src.getCar().getId(), PaymentDto::setCarId));
         typeMap.addMappings(mapper -> mapper.map(src -> src.getUser().getId(), PaymentDto::setUserID));
-        typeMap.addMapping(Payment::getDate, PaymentDto::setDate);
+
+    }
+
+    private void createTypeMapperDtoToEntity() {
+        TypeMap<PaymentDto, Payment> typeMap = modelMapper.createTypeMap(PaymentDto.class, Payment.class);
+        typeMap.addMapping(PaymentDto::getAmount, Payment::setAmount);
```

Comment:
You can move this logic to AppConfig where we instantiate our modelMaper object. This will add them only when we create the modelMapper and not on every request.